lib/components/nav/navHeader.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:html';
```

Comment:
can we rename this file to `nav_header.dart`? camel case is not very standard for file names